docs/application/dotnet/guides/migration/migrate-xaml-android.md

```diff
@@ -0,0 +1,7 @@
+# Migrate Xamarin.Android App to Xamarin.Tizen
+
```

Comment:
This file still does not have information, and so cannot proceed further with this PR review.

Reply:
@patrykka, is this file part of this PR